Debes añadir al fichero oca_dependencies.txt una línea

```
odoomrp-wip https://github.com/odoomrp/odoomrp-wip
```

Ya que intentas usar un modulo que está en ella, y por ello fallan los tests de travis.

Además no sé como lo has hecho pero los errores de Pylint se deben a que estas subiendo los ficheros con permisos de ejecución.

y el PEP8 falla porque:

```
mrp_repair_fee_menu/models/mrp_repair.py:1:2: E111 indentation is not a multiple of four
mrp_repair_fee_menu/models/mrp_repair.py:1:2: E113 unexpected indentation
```

--- a/mrp_repair_fee_menu/README.rst
+++ b/mrp_repair_fee_menu/README.rst
@@ -1,0 +1,21 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+MRP repair fee menu
+=======================
+
+* This module creates the new menu option "Repair Fees Lines".
+* In "Repair Fees Lines", the cost price of the product is also shown.
+* When creating a new line of "Repair Fees Lines", default as a product, he
+  will catch the defined in the employee associated with the user.
+
+Credits
+=======
+
+Contributors
+------------
+* Ana Juaristi <anajuaristi@avanzosc.es>
+* Oihane Crucelaegui <oihanecrucelaegi@avanzosc.es>
+* Alfredo de la Fuente <alfredodelafuente@avanzosc.es>

--- a/mrp_repair_fee_menu/__init__.py
+++ b/mrp_repair_fee_menu/__init__.py
@@ -1,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Alfredo de la Fuente - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from . import models

--- a/mrp_repair_fee_menu/__openerp__.py
+++ b/mrp_repair_fee_menu/__openerp__.py
@@ -1,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Alfredo de la Fuente - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+{
+    'name': 'MRP Repair Fee Menu',
+    "version": "8.0.1.0.0",
+    "license": 'AGPL-3',
+    "author": 'OdooMRP team,'
+              'AvanzOSC,'
+              'Serv. Tecnol. Avanzados - Pedro M. Baeza',
+    'website': "http://www.odoomrp.com",
+    "contributors": [
+        "Ana Juaristi <anajuaristi@avanzosc.es>",
+        "Oihane Crucelaegui <oihanecrucelaegi@avanzosc.es>",
+        "Alfredo de la Fuente <alfredodelafuente@avanzosc.es>",
+        ],
+    'category': 'Manufacturing',
+    'depends': ['base',
+                'product',
+                'stock',
+                'hr',
+                'hr_timesheet',
+                'mrp_repair',
+                'mrp_repair_analytic'
+                ],
+    'data': ['views/mrp_repair_view.xml',
+             'views/mrp_repair_fee_view.xml',
+             ],
+    'installable': True,
+}

--- a/mrp_repair_fee_menu/i18n/es.po
+++ b/mrp_repair_fee_menu/i18n/es.po
@@ -1,0 +1,156 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* mrp_repair_fee_menu
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 8.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2015-11-20 11:45+0000\n"
+"PO-Revision-Date: 2015-11-20 12:46+0100\n"
+"Last-Translator: <>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: \n"
+"X-Generator: Poedit 1.5.4\n"
+
+#. module: mrp_repair_fee_menu
+#: help:mrp.repair.fee,repair_partner:0
+msgid "Choose partner for whom the order will be invoiced and delivered."
+msgstr "Elija empresa para quien se facturará el pedido y entrega."
+
+#. module: mrp_repair_fee_menu
+#: view:mrp.repair:mrp_repair_fee_menu.view_repair_order_form_feemenu
+msgid "Fees"
+msgstr "Cargos"
+
+#. module: mrp_repair_fee_menu
+#: view:mrp.repair:mrp_repair_fee_menu.view_repair_order_form_feemenu
+#: field:mrp.repair,fees_lines_no_to_invoice:0
+msgid "Fees to NOT invoice"
+msgstr "Cargos a NO facturar"
+
+#. module: mrp_repair_fee_menu
+#: view:mrp.repair:mrp_repair_fee_menu.view_repair_order_form_feemenu
+#: field:mrp.repair,fees_lines_to_invoice:0
+msgid "Fees to invoice"
+msgstr "Cargos a facturar"
+
+#. module: mrp_repair_fee_menu
+#: view:mrp.repair.fee:mrp_repair_fee_menu.mrp_repair_fee_search_view
+msgid "Group By"
+msgstr "Agrupar por"
+
+#. module: mrp_repair_fee_menu
+#: view:mrp.repair:mrp_repair_fee_menu.view_repair_order_form_feemenu
+#: view:mrp.repair.fee:mrp_repair_fee_menu.mrp_repair_fee_form_view
+msgid "History"
+msgstr "Historial"
+
+#. module: mrp_repair_fee_menu
+#: field:mrp.repair.fee,repair_partner:0
+msgid "Partner"
+msgstr "Empresa"
+
+#. module: mrp_repair_fee_menu
+#: view:mrp.repair.fee:mrp_repair_fee_menu.mrp_repair_fee_search_view
+msgid "Product Fee"
+msgstr "Producto cargo"
+
+#. module: mrp_repair_fee_menu
+#: view:mrp.repair:mrp_repair_fee_menu.view_repair_order_form_feemenu
+#: view:mrp.repair.fee:mrp_repair_fee_menu.mrp_repair_fee_form_view
+msgid "Product Information"
+msgstr "Información del producto"
+
+#. module: mrp_repair_fee_menu
+#: field:mrp.repair.fee,repair_product:0
+msgid "Product To Repair"
+msgstr "Producto a reparar"
+
+#. module: mrp_repair_fee_menu
+#: help:mrp.repair.fee,repair_lot:0
+msgid "Products repaired are all belonging to this lot"
+msgstr "Los productos reparados son todos ellos pertenecientes a este lote"
+
+#. module: mrp_repair_fee_menu
+#: view:mrp.repair:mrp_repair_fee_menu.view_repair_order_form_feemenu
+#: view:mrp.repair.fee:mrp_repair_fee_menu.mrp_repair_fee_form_view
+#: view:mrp.repair.fee:mrp_repair_fee_menu.mrp_repair_fee_tree_view
+msgid "Quantity"
+msgstr "Cantidad"
+
+#. module: mrp_repair_fee_menu
+#: view:mrp.repair.fee:mrp_repair_fee_menu.mrp_repair_fee_search_view
+msgid "Repair"
+msgstr "Reparación"
+
+#. module: mrp_repair_fee_menu
+#: model:ir.model,name:mrp_repair_fee_menu.model_mrp_repair_fee
+#: view:mrp.repair.fee:mrp_repair_fee_menu.mrp_repair_fee_form_view
+msgid "Repair Fees Line"
+msgstr "Línea cargo reparación"
+
+#. module: mrp_repair_fee_menu
+#: model:ir.actions.act_window,name:mrp_repair_fee_menu.action_mrp_repair_fee
+#: model:ir.ui.menu,name:mrp_repair_fee_menu.mrp_repair_fee_menu_view
+#: view:mrp.repair.fee:mrp_repair_fee_menu.mrp_repair_fee_tree_view
+msgid "Repair Fees Lines"
+msgstr "Líneas cargo reparación"
+
+#. module: mrp_repair_fee_menu
+#: view:mrp.repair.fee:mrp_repair_fee_menu.mrp_repair_fee_search_view
+#: field:mrp.repair.fee,repair_lot:0
+msgid "Repair Lot"
+msgstr "Lote reparación"
+
+#. module: mrp_repair_fee_menu
+#: model:ir.model,name:mrp_repair_fee_menu.model_mrp_repair
+msgid "Repair Order"
+msgstr "Orden reparación"
+
+#. module: mrp_repair_fee_menu
+#: view:mrp.repair.fee:mrp_repair_fee_menu.mrp_repair_fee_search_view
+msgid "Repair Partner"
+msgstr "Reparación empresa"
+
+#. module: mrp_repair_fee_menu
+#: view:mrp.repair.fee:mrp_repair_fee_menu.mrp_repair_fee_search_view
+msgid "Repair Product"
+msgstr "Reparación producto"
+
+#. module: mrp_repair_fee_menu
+#: view:mrp.repair.fee:mrp_repair_fee_menu.mrp_repair_fee_search_view
+msgid "Search Repair Fees Lines"
+msgstr "Buscar líneas de cargo de reparación"
+
+#. module: mrp_repair_fee_menu
+#: code:addons/mrp_repair_fee_menu/models/mrp_repair.py:54
+#, python-format
+msgid "The employee associated with the user has not defined any product"
+msgstr "El empleado asociado al usuario, no tiene definido ningún producto"
+
+#. module: mrp_repair_fee_menu
+#: view:mrp.repair:mrp_repair_fee_menu.view_repair_order_form_feemenu
+#: view:mrp.repair.fee:mrp_repair_fee_menu.mrp_repair_fee_tree_view
+msgid "Unit of Measure"
+msgstr "Unidad de medida"
+
+#. module: mrp_repair_fee_menu
+#: view:mrp.repair.fee:mrp_repair_fee_menu.mrp_repair_fee_search_view
+msgid "User"
+msgstr "Usuario"
+
+#. module: mrp_repair_fee_menu
+#: code:addons/mrp_repair_fee_menu/models/mrp_repair.py:51
+#, python-format
+msgid "User does not have any employee assigned"
+msgstr "Usuario no está asignado a ningún empleado"
+
+#. module: mrp_repair_fee_menu
+#: code:addons/mrp_repair_fee_menu/models/mrp_repair.py:49
+#, python-format
+msgid "Warning!"
+msgstr "Aviso!"

--- a/mrp_repair_fee_menu/models/__init__.py
+++ b/mrp_repair_fee_menu/models/__init__.py
@@ -1,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Alfredo de la Fuente - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from . import mrp_repair

--- a/mrp_repair_fee_menu/models/mrp_repair.py
+++ b/mrp_repair_fee_menu/models/mrp_repair.py
@@ -1,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Alfredo de la Fuente - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from openerp import models, fields, api, _
+
+
+class MrpRepair(models.Model):
+    _inherit = 'mrp.repair'
+
+    fees_lines_to_invoice = fields.One2many(
+        'mrp.repair.fee', 'repair_id', string='Fees to invoice',
+        readonly=True, states={'draft': [('readonly', False)]}, copy=True,
+        domain=[('to_invoice', '=', True)])
+    fees_lines_no_to_invoice = fields.One2many(
+        'mrp.repair.fee', 'repair_id', string='Fees to NOT invoice',
+        readonly=True, states={'draft': [('readonly', False)]}, copy=True,
+        domain=[('to_invoice', '=', False)])
+
+
+class MrpRepairFee(models.Model):
+    _inherit = 'mrp.repair.fee'
+
+    to_invoice = fields.Boolean(default=False)
+    standard_price = fields.Float(
+        string='Cost Price', related='product_id.standard_price')
+    repair_pricelist = fields.Many2one(
+        related='repair_id.pricelist_id', string='Pricelist')
+    repair_partner = fields.Many2one(
+        related='repair_id.partner_id', string='Partner', store=True)
+    repair_product = fields.Many2one(
+        related='repair_id.product_id', string='Product To Repair',
+        store=True)
+    repair_lot = fields.Many2one(
+        related='repair_id.lot_id', string='Repair Lot', store=True)
+    repair_guarantee_limit = fields.Date(
+        related='repair_id.guarantee_limit', string='Warranty Expiration')
+
+    @api.multi
+    @api.onchange('user_id')
+    def _onchange_user_id(self):
+        employee_obj = self.env['hr.employee']
+        self.ensure_one()
+        result = {}
+        cond = [('user_id', '=', self.user_id.id)]
+        employee = employee_obj.search(cond, limit=1)
+        if employee and employee.product_id:
+            self.product_id = employee.product_id.id
+        else:
+            warning = {'title': _('Warning!')}
+            if not employee:
+                warning['message'] = _('User does not have any employee '
+                                       'assigned')
+            elif not employee.product_id:
+                warning['message'] = _('The employee associated with the user'
+                                       ' has not defined any product')
+            self.product_id = False
+            self.name = False
+            result['warning'] = warning
+        return result

--- a/mrp_repair_fee_menu/tests/__init__.py
+++ b/mrp_repair_fee_menu/tests/__init__.py
@@ -1,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Alfredo de la Fuente - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from . import test_mrp_repair_fee_menu

--- a/mrp_repair_fee_menu/tests/test_mrp_repair_fee_menu.py
+++ b/mrp_repair_fee_menu/tests/test_mrp_repair_fee_menu.py
@@ -1,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Alfredo de la Fuente - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+import openerp.tests.common as common
+
+
+class TestMrpFeeMenu(common.TransactionCase):
+
+    def setUp(self):
+        super(TestMrpFeeMenu, self).setUp()
+        self.data_model = self.env['ir.model.data']
+        fee_vals = {'user_id': self.env.ref('base.user_root').id,
+                    'name': 'Fee line test',
+                    'product_uom': self.env.ref('product.product_uom_unit').id,
+                    'price_unit': 1,
+                    'product_uom_qty': 5}
+        vals = {'product_id': self.env.ref('product.product_product_8').id,
+                'product_uom': self.env.ref('product.product_uom_unit').id,
+                'location_id': self.env.ref('stock.stock_location_7').id,
+                'location_dest_id': self.env.ref('stock.stock_location_7').id,
+                'fees_lines': [(0, 0, fee_vals)]}
+        self.repair = self.env['mrp.repair'].create(vals)
+
+    def test_mrp_repair_fee(self):
+        self.assertEqual(
+            len(self.repair.fees_lines_no_to_invoice), 1,
+            'Repair without fee to not invoice')
+        self.repair.fees_lines_no_to_invoice[0]._onchange_user_id()
+        self.assertEqual(
+            self.repair.fees_lines_no_to_invoice[0].product_id.id,
+            self.env.ref('product.product_product_consultant').id,
+            'Wrong Product for the administrator user')

--- a/mrp_repair_fee_menu/views/mrp_repair_fee_view.xml
+++ b/mrp_repair_fee_menu/views/mrp_repair_fee_view.xml
@@ -1,0 +1,106 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record model="ir.ui.view" id="mrp_repair_fee_search_view">
+            <field name="name">mrp.repair.fee.search.view</field>
+            <field name="model">mrp.repair.fee</field>
+            <field name="arch" type="xml">
+                <search string="Search Repair Fees Lines">
+                    <field name="user_id" />
+                    <field name="repair_partner" />
+                    <field name="repair_product" />
+                    <field name="product_id" />
+                    <field name="repair_id" />
+                    <field name="repair_lot" />
+                    <field name="name" />
+                    <group expand="0" string="Group By">
+                        <filter string="User" domain="[]" context="{'group_by':'user_id'}"/>
+                        <filter string="Repair Partner" domain="[]" context="{'group_by':'repair_partner'}"/>
+                        <filter string="Repair Product" domain="[]" context="{'group_by':'repair_product'}"/>
+                        <filter string="Product Fee" domain="[]" context="{'group_by':'product_id'}"/>
+                        <filter string="Repair" domain="[]" context="{'group_by':'repair_id'}"/>
+                        <filter string="Repair Lot" domain="[]" context="{'group_by':'repair_lot'}"/>
+                    </group>
+                </search>
+            </field>
+        </record>
+        <record model="ir.ui.view" id="mrp_repair_fee_tree_view" >
+            <field name="name">mrp.repair.fee.search.view</field>
+            <field name="model">mrp.repair.fee</field>
+            <field name="arch" type="xml">
+                <tree string="Repair Fees Lines">
+                    <field name="user_id"/>
+                    <field name="repair_id" />
+                    <field name="repair_partner" />
+                    <field name="repair_product" />
+                    <field name="repair_lot" />
+                    <field name="product_id"/>
+                    <field name='name'/>
+                    <field name="product_uom_qty" string="Quantity"/>
+                    <field name="product_uom" string="Unit of Measure" groups="product.group_uom"/>
+                    <field name="price_unit"/>
+                    <field name="standard_price" />
+                    <field name="tax_id"/>
+                    <field name="to_invoice"/>
+                    <field name="price_subtotal"/>
+                </tree>
+            </field>
+        </record>
+
+        <record model="ir.ui.view" id="mrp_repair_fee_form_view" >
+            <field name="name">mrp.repair.fee.form.view</field>
+            <field name="model">mrp.repair.fee</field>
+            <field name="arch" type="xml">
+                <form string="Repair Fees Line">
+                    <label for="name" class="oe_edit_only"/>
+                    <h2>
+                        <field name="name"/>
+                    </h2>
+                    <group string="Product Information">
+                        <group>
+                            <field name="repair_id" />
+                            <field name="repair_pricelist" invisible="1"/>
+                            <field name="repair_partner" invisible="1"/>
+                            <field name="repair_product" invisible="1" />
+                            <field name="repair_lot" invisible="1" />
+                            <field name="repair_guarantee_limit" invisible="1"/>
+                        </group>
+                        <group>
+                            <field name="user_id"/>
+                            <field name="to_invoice"/>
+                            <field name="product_id" on_change="product_id_change(repair_pricelist, product_id, product_uom, product_uom_qty, repair_partner, repair_guarantee_limit, context)"/>
+                            <label for="product_uom_qty"/>
+                            <div>
+                                <field name="product_uom_qty" string="Quantity" class="oe_inline"/>
+                                <field name="product_uom" groups="product.group_uom" class="oe_inline"/>
+                            </div>
+                        </group>
+                        <group>
+                            <field name="price_unit"/>
+                            <field name="standard_price" />
+                            <field widget="many2many_tags" name="tax_id" domain="[('parent_id','=',False),('type_tax_use','&lt;&gt;','purchase')]"/>
+                            <field name="price_subtotal"/>
+                        </group>
+                    </group>
+                    <group string="History">
+                        <group>
+                            <field name="invoiced"/>
+                        </group>
+                        <group>
+                            <field name="invoice_line_id" />
+                        </group>
+                    </group>
+                </form>
+            </field>
+        </record>
+
+        <record model="ir.actions.act_window" id="action_mrp_repair_fee">
+            <field name="name">Repair Fees Lines</field>
+            <field name="res_model">mrp.repair.fee</field>
+            <field name="view_type">form</field>
+            <field name="view_mode">tree,form</field>
+        </record>
+        <menuitem id="mrp_repair_fee_menu_view" name="Repair Fees Lines"
+            parent="mrp.menu_mrp_manufacturing" action="action_mrp_repair_fee" sequence="53"/>
+    </data>
+</openerp>

--- a/mrp_repair_fee_menu/views/mrp_repair_view.xml
+++ b/mrp_repair_fee_menu/views/mrp_repair_view.xml
@@ -1,0 +1,114 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record id="view_repair_order_form_feemenu" model="ir.ui.view">
+            <field name="name">view.repair.order.form.feemenu</field>
+            <field name="model">mrp.repair</field>
+            <field name="inherit_id" ref="mrp_repair.view_repair_order_form" />
+            <field name="arch" type="xml">
+                <separator string="Fees" position="attributes">
+                    <attribute name="invisible">1</attribute>
+                </separator>
+                <field name="fees_lines" position="attributes">
+                    <attribute name="invisible">1</attribute>
+                </field>
+                <field name="fees_lines" position="after">
+                    <separator string="Fees to invoice" />
+                    <field name="fees_lines_to_invoice" attrs="{'readonly': [('invoice_method','=', 'none')]}">
+                        <form string="Fees to invoice">
+                            <label for="name" class="oe_edit_only"/>
+                            <h2>
+                                <field name="name"/>
+                            </h2>
+                            <group string="Product Information">
+                                <group>
+                                    <field name="user_id"/>
+                                    <field name="to_invoice"/>
+                                    <field name="product_id" on_change="product_id_change(parent.pricelist_id, product_id, product_uom, product_uom_qty, parent.partner_id, parent.guarantee_limit, context)"/>
+                                    <label for="product_uom_qty"/>
+                                    <div>
+                                        <field name="product_uom_qty" string="Quantity" class="oe_inline"/>
+                                        <field name="product_uom" groups="product.group_uom" class="oe_inline"/>
+                                    </div>
+                                </group>
+                                <group>
+                                    <field name="price_unit"/>
+                                    <field name="standard_price" />
+                                    <field widget="many2many_tags" name="tax_id" domain="[('parent_id','=',False),('type_tax_use','&lt;&gt;','purchase')]"/>
+                                    <field name="price_subtotal"/>
+                                </group>
+                            </group>
+                            <group string="History">
+                                <group>
+                                    <field name="invoiced"/>
+                                </group>
+                                <group>
+                                    <field name="invoice_line_id" />
+                                </group>
+                            </group>
+                        </form>
+                        <tree string="Fees" editable="bottom">
+                            <field name="user_id"/>
+                            <field name="product_id" on_change="product_id_change(parent.pricelist_id, product_id, product_uom, product_uom_qty, parent.partner_id,parent.guarantee_limit, context)"/>
+                            <field name='name'/>
+                            <field name="product_uom_qty" string="Quantity"/>
+                            <field name="product_uom" string="Unit of Measure" groups="product.group_uom"/>
+                            <field name="price_unit"/>
+                            <field name="standard_price" />
+                            <field name="tax_id" domain="[('parent_id', '=', False), ('type_tax_use', '=', 'purchase')]" widget="many2many_tags"/>
+                            <field name="to_invoice"/>
+                            <field name="price_subtotal"/>
+                        </tree>
+                    </field>
+                    <separator string="Fees to NOT invoice" />
+                    <field name="fees_lines_no_to_invoice" attrs="{'readonly': [('invoice_method','=', 'none')]}">
+                        <form string="Fees">
+                            <label for="name" class="oe_edit_only"/>
+                            <h2>
+                                <field name="name"/>
+                            </h2>
+                            <group string="Product Information">
+                                <group>
+                                    <field name="user_id"/>
+                                    <field name="to_invoice"/>
+                                    <field name="product_id" on_change="product_id_change(parent.pricelist_id, product_id, product_uom, product_uom_qty, parent.partner_id, parent.guarantee_limit, context)"/>
+                                    <label for="product_uom_qty"/>
+                                    <div>
+                                        <field name="product_uom_qty" string="Quantity" class="oe_inline"/>
+                                        <field name="product_uom" groups="product.group_uom" class="oe_inline"/>
+                                    </div>
+                                </group>
+                                <group>
+                                    <field name="price_unit"/>
+                                    <field name="standard_price" />
+                                    <field widget="many2many_tags" name="tax_id" domain="[('parent_id','=',False),('type_tax_use','&lt;&gt;','purchase')]"/>
+                                    <field name="price_subtotal"/>
+                                </group>
+                            </group>
+                            <group string="History">
+                                <group>
+                                    <field name="invoiced"/>
+                                </group>
+                                <group>
+                                    <field name="invoice_line_id" />
+                                </group>
+                            </group>
+                        </form>
+                        <tree string="Fees" editable="bottom">
+                            <field name="user_id"/>
+                            <field name="product_id" on_change="product_id_change(parent.pricelist_id, product_id, product_uom, product_uom_qty, parent.partner_id,parent.guarantee_limit, context)"/>
+                            <field name='name'/>
+                            <field name="product_uom_qty" string="Quantity"/>
+                            <field name="product_uom" string="Unit of Measure" groups="product.group_uom"/>
+                            <field name="price_unit"/>
+                            <field name="standard_price" />
+                            <field name="tax_id" domain="[('parent_id', '=', False), ('type_tax_use', '=', 'purchase')]" widget="many2many_tags"/>
+                            <field name="to_invoice"/>
+                            <field name="price_subtotal"/>
+                        </tree>
+                    </field>
+                </field>
+            </field>
+        </record>
+    </data>
+</openerp>

--- a/oca_dependencies.txt
+++ b/oca_dependencies.txt
@@ -1,5 +1,6 @@
 # List the OdooMRP project dependencies, one per line
-
+odoomrp-wip https://github.com/odoomrp/odoomrp-wip
 # List the OCA project dependencies, one per line
 
 # Add a repository url and branch if you need a forked version
+

